src/installer/pkg/projects/Directory.Build.targets

```diff
@@ -19,7 +19,7 @@
     out and skip building. 
   -->
   <Target Name="MobileSkipBuildProps" 
-          Condition="'$(SkipBuildOnRuntimePackOnlyOS)' == 'true' and ('$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS' or '$(TargetOS)' == 'Android')"
+          Condition="'$(SkipBuildOnRuntimePackOnlyOS)' == 'true' and '$(TargetsMobile)' == 'true'"
```

Comment:
Leave this one as is, this is taking care here: https://github.com/dotnet/runtime/pull/34980/files#diff-0a57cd77a878d620b1edc73b4dcc9f1fR22

Reply:
Thanks, fixed merge conflicts